ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -74,7 +74,7 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     # Set-up environment variables.
     export NNST_ROOT="${dir_ci}/${dir_commit}/${PRJ_REPO_OWNER}"
     export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$NNST_ROOT/lib
-    export GST_PLUGIN_PATH=$GST_PLUGIN_PATH:$NNST_ROOT/lib
+    export GST_PLUGIN_PATH=$GST_PLUGIN_PATH:$NNST_ROOT/lib/gstreamer-1.0
```

Comment:
> After refactoring the CMakeLists.txt files in the nnstreamer repository, 

Please tell me the PR number. 
Note that apptest is executed  in the  git repository folder (without the system partition of Ubuntu OS).

Reply:
[FYI] Currently, I could not find the "..../lib/gstreamer-1/0" fole namer  with nnstreamer's PR  900.
* http://nnsuite.mooo.com/nnstreamer/ci/repo-workers/pr-checker/900-201811261411400.44143104553223-e00b52378de0b0499543170d7726c15a1bf49393/nnstreamer/lib/

Reply:
@leemgs Please refer the following commits:
https://github.com/nnsuite/nnstreamer/commit/47740561481ed20bef355f80315438935357521c
https://github.com/nnsuite/nnstreamer/commit/1ff3440c1bf5e1d9a6a2849ed7f8d1e6332a561b

Reply:
Got it. Synced.